keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -582,6 +814,13 @@ class SqlArtifactRepository(
       .where(ENVIRONMENT.NAME.eq(environment.name))
       .and(ENVIRONMENT.DELIVERY_CONFIG_UID.eq(uid))
 
+  private fun DeliveryConfig.getUidStringFor(environment: Environment): String =
```

Comment:
Side-comment: I find these methods named `SomeClass.getSomethingFor(another: AnotherClass)` hard to read. I'd suggest adding the name of the thing we're getting the property fo r, as in `DeliveryConfig.getUidStringForEnvironment(environment)`, in this case. I know it makes for more verbose code, but also more readable IMHO.

Beyond the scope of this PR, though, as they're all over the place.

Reply:
What do you mean? Inellij can show you the type hints if the actual type is not present.

Reply:
Discovering intellj's command-p feature while typing a method (i.e.: `foo(` <- hit cmd-p here)) after a year of use was a revelation for me in case it's new to you as well.